src/lib/parameterize.ts

```diff
@@ -1,23 +1,34 @@
 import {logger} from '../util/logger';
 import {memoizedLog} from '../util/log-memoize';
+import {debugLogger} from '../util/debug-logger';
 
 import {STRINGS, PARAMETERS} from '../config';
 
 import {ManifestParameter} from '../types/manifest';
 import {Parameters} from '../types/parameters';
 
-const {REJECTING_OVERRIDE, UNKNOWN_PARAM} = STRINGS;
+const {
+  REJECTING_OVERRIDE,
+  UNKNOWN_PARAM,
+  SYNCING_PARAMETERS,
+  // CHECKING_AGGREGATION_METHOD,
```

Comment:
?

Reply:
thanks for noticing. I forgot uncomment before push